unicode-data-names/unicode-data-names.cabal

```diff
@@ -85,6 +90,36 @@ library
   hs-source-dirs: lib
   build-depends:
       base             >= 4.7   && < 4.19,
+  -- Support for raw string literals unpacking is included in base ≥ 4.15
+  if impl(ghc < 9.0.0)
+    build-depends:
+      ghc-prim >= 0.3.1 && < 1.0
+
+-- Internal library to enable comparison to ICU
+library icu-names
+  import: default-extensions, compile-options
+  if flag(has-icu)
+    buildable: True
+  else
+    buildable: False
+  exposed-modules:
+    ICU.Names
+  hs-source-dirs: lib
+  build-depends:
+      base             >= 4.7   && < 4.19
+  include-dirs: cbits
+  c-sources: cbits/icu.c
+  cc-options: -Wall -Wextra -pedantic
+  if os(darwin)
+    extra-lib-dirs:
+      /usr/local/opt/icu4c/lib
+      /opt/homebrew/opt/icu4c/lib
+    include-dirs:
+      /usr/local/opt/icu4c/include
+      /opt/homebrew/opt/icu4c/include
```

Comment:
Non-standard paths should not be hardcoded here. We can use cabal.project to specify these paths.

Reply:
I agree. Those are taken from the `text-icu` package, I do not know what use instead. Should I remove them completely and mark it as non buildable for this platform?

Reply:
We can leave it buildable, if the library is found in standard path it might build, if not the onus is on the user to provide the installation and use cabal options to specify where to find it, we can add these as suggested paths in the README.

Reply:
Ok. I moved `icu-names` to its own (internal) package. In fact there is no reason to ship it with `unicode-data-names`. It is really only an internal package, enabling further tests.